NuRadioMC/simulation/simulation.py

```diff
@@ -576,6 +577,7 @@ def run(self):
 #                     input_time += (time.time() - t1)
 
                     for channel_id in range(self._det.get_number_of_channels(self._station_id)):
+                        self._raytracer.reset_solutions()
```

Comment:
Not sure anymore why i moved it out, maybe because I found it tricky to reset everything when redefining the start and end point because you loose then your previous solutions. In the analytical tracer is may not be a very big deal because it is fast but in radiopropa this might cause extra computing time like. this should be discussed maybe

Reply:
@cg-laser what do you think?

Reply:
In the analytic RT the reset is still done in `set_start_and_end_point` but I removed the reset out of this function in the radiopropa RT as the calculation of solution is quiet abour intensive and I thought this way people sill not reset the solution by accident if they wanted to do something else in the meantime by setting different start and end point. Not sure if this makes sense

Reply:
done